core/src/autogluon/core/models/ensemble/weighted_ensemble_model.py

```diff
@@ -72,3 +72,8 @@ def _set_default_params(self):
         for param, val in default_params.items():
             self._set_default_param_value(param, val)
         super()._set_default_params()
+
+    def compile(self, path: str = None, verbose=True, compiler_configs={}) -> str:
```

Comment:
Don't use `{}` as default (mutable), use `None` and then `if compiler_configs is None: compiler_configs = {}`

Reply:
Done.